src/sass/themes/donate/components/form/_form.scss

```diff
@@ -179,7 +179,7 @@ button.payment-button {
 	// fallback if apple pay styling doesn't come through
 	// making sure this doesn't stick out if apple styling does come through
 	height: 55px;
-	@extend button.cta;
+  @extend button, .cta;

```

Comment:
Not sure what's going on with the spacing/tab changes here but I can't fix it either soooo 🤷